eventhub/Module.md

```diff
@@ -127,17 +105,17 @@ You can specify the event hub path and the event data as parameters of the send
 This operation will return a ballerina error if the operation failed.
 
 
-## Entity Management in an Azure Event Hub 
+### Entity Management in an Azure Event Hub 
 This is the simplest scenario to manage entities related to azure event hubs. You need to obtain a connection string of 
```

Comment:
Again is this a prerequisite?

Reply:
Please apply the above review comments to this section as well.

Reply:
Fixed in df7c75b . Thank you.